### ISSUE TITLE
[REQUEST] Add a setting to hide thumbnails/previews

### ISSUE THREAD

**Issue by DavrK** [Stremio/stremio-features#775]
**Stremio Version**
All Stremio versions

**Is your feature request related to a problem? Please describe.**
Not necessarily a problem, just something that would make the app better

**Describe the solution you'd like**
To have a simple setting that allows the user to hide the thumbnails/previews of streams

**Describe alternatives you've considered**
Tried to look for an addon that adds this feature but didn't find any

**Additional context**
To avoid spoilers

**Comment by byrek3d** [comment]
Agreed, this is a must have. Ideally the thumbnails should become visible only after you have watched an episode, but as an alternative having an option that completely removed thumbnails would be already very very useful

**Comment by GxDeWitt** [comment]
I agree, is necessary to avoid spoilers.

**Comment by Allebale** [comment]
I would like to have this too

**Comment by Blixtmen** [comment]
Yes would be a great addition 👍

**Comment by timurkyt** [comment]
https://github.com/Stremio/stremio-features/issues/504

I think they are working on it because they haven't closed the issue since half a year ago.

**Comment by i1u5** [comment]
+1

This is a problem especially in old tv shows where images fetched through TMDB show potential spoilers like to-be/presumed dead characters are shown in future episode thumbnails. The new shows seem to use the official ones from streaming services like Netflix so it's a bit rare, but still possible because TMDB images are community managed.

I think another good addition would be the option to hide the background image too, leaving only the "logos" visible, an example where this is a problem is the tv show House M.D where there is an episode (S02E20) where a character almost dies but in the background (the one in the main page, not the thumbnail) you can see him older which means it's fetched from later seasons, meaning he survives (which he does an episode later).

**Comment by mjurcy** [comment]
Totally agree with everything mentioned here. Having the option to hide thumbnails would be really helpful to avoid spoilers, especially for older shows where TMDB images can reveal key moments. I also think hiding the background while keeping only the logos visible is a great idea. Hopefully, this gets implemented soon.

**Comment by mesalilac** [comment]
I found a setting called `Blur unwatched episodes image`, but it only appears in the browser and applies only to the browser version, no desktop or mobile support

UPDATE: You will need to download the v5 version of the desktop app to have the `Blur unwatched episodes image` setting

**Comment by kKaskak** [comment]
this setting is widely supported now so im closing this as completed you can find it under 
settings -> Blur unwatched episodes image
